datafusion/sqllogictest/test_files/datetime/arith_interval_double.slt

```diff
@@ -0,0 +1,31 @@
+# interval * double precision → interval
```

Comment:
Maybe add a comment that those are not supported yet ?!
There are only error checks below.